{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings.py

```diff
@@ -113,6 +113,23 @@
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 SSLIFY_DISABLE = config('SSLIFY_DISABLE', default=False, cast=bool)
 
+{% if cookiecutter.use_redis == "y" -%}
+
+# redis
+
+SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
+CACHES = {
+    "default": {
```

Comment:
Please change double quotes to single quotes to maintain consistency with the rest of the file.